azurerm/internal/services/databricks/resource_arm_databricks_workspace.go

```diff
@@ -69,6 +69,32 @@ func resourceArmDatabricksWorkspace() *schema.Resource {
 				ValidateFunc: validate.NoEmptyStrings,
 			},
 
+			"workspace_custom_parameters": {
```

Comment:
_minor_ could we make this:

```suggestion
			"custom_parameters": {
```

Reply:
Updated